test/lib/render-helpers-navigate.js

```diff
@@ -40,17 +58,23 @@ I18nProvider.defaultProps = {
   children: undefined,
 };
 
-function createProviderWrapper(store, pathname = '/') {
+function createProviderWrapper(
+  store,
+  pathname = '/',
+  getMockTrackEvent = () => () => Promise.resolve(),
```

Comment:
Could we also update these arguments in this file to align with the changes to `createMockTrackEvent`?

```suggestion
  getMockTrackEvent = () => jest.fn().mockResolvedValue(undefined),
```

Reply:
yes of course !! good idea ~~ addressed in 3bed7f7c46668dc7ca0e0ada1f7f1e321d5bab0f